### DISCUSSION TITLE
Clarification

### DISCUSSION THREAD

**Discussion by MrBen1909** [winglessraven/AMP-Discord-Bot#105] [General]
Do I have to setup a bot for each game? Or can I use the same bot for different games?

**Comment by winglessraven** [comment]
It's one bot per game, by design. That way it's easier to see each game server's status in the online list...
 
![image](https://github.com/user-attachments/assets/6688a751-966b-4e88-9a97-41da573a537d)

**Comment by Bats92** [comment]
Sorry to dig up an old discussion - does this mean we need multiple developer licenses as it is one per bot?

**Reply by winglessraven** [reply]
> Sorry to dig up an old discussion - does this mean we need multiple developer licenses as it is one per bot?

One dev license will cover unlimited instances. Cubecoders have confirmed that the limit of 1 shown in the license portal is not applicable
